app/charts/area/areas-state.tsx

```diff
@@ -372,18 +376,21 @@ const useAreasState = (
         formatNumber,
       });
       const xAnchor = xScale(getX(datum));
-      const yAnchor = normalize
+      const yNormalized = normalize
```

Comment:
Nit: The name of the variable is only correct in "half" of the cases (when `normalize` is true) 😄 Maybe it could be renamed to `yDesktopAnchor` to make a connection with how it's used afterwards?